src/Algebra/Properties/Group.agda

```diff
@@ -127,6 +127,19 @@ x≈y⇒x∙y⁻¹≈ε {x} {y} x≈y = begin
   y ∙ y ⁻¹ ≈⟨ inverseʳ y ⟩
   ε        ∎
 
+x≈ε⇒x⁻¹≈ε : ∀ {x} → x ≈ ε → x ⁻¹ ≈ ε
+x≈ε⇒x⁻¹≈ε {x} x≈ε = begin
+  x ⁻¹   ≈⟨ ⁻¹-cong x≈ε ⟩
+  ε ⁻¹   ≈⟨ ε⁻¹≈ε ⟩
+  ε ∎
```

Comment:
I wasn't sure if
* this lemma met the Fairbairn threshhold
* this proof was the most 'efficient', compared to
  ```agda
  x≈ε⇒x⁻¹≈ε = identityˡ-unique _ _ ∘ trans (inverseˡ _) ∘ sym
  ```
which uses only a `Loop` property beyond the basic `IsGroup` axiomatisation.